pyls/plugins/hover.py

```diff
@@ -13,6 +13,15 @@ def pyls_hover(document, position):
     # Find an exact match for a completion
     for d in definitions:
         if d.name == word:
-            return {'contents': _utils.format_docstring(d.docstring()) or ''}
+            doc = _utils.format_docstring(d.docstring()).split('\n\n', 1)
```

Comment:
Why is this `split('\n\n', 1)` is needed?

Reply:
so I can separate signature from docstring